metadata-jobs/mce-consumer-job/build.gradle

```diff
@@ -4,14 +4,34 @@ plugins {
 }
 
 dependencies {
-    compile project(':metadata-jobs:mce-consumer')
-    compile(externalDependency.springBootStarterWeb) {
+    implementation project(':metadata-service:factories')
+    implementation project(':metadata-jobs:mce-consumer')
+    implementation project(':entity-registry')
+
+    implementation(externalDependency.springBootStarterWeb) {
         exclude module: "spring-boot-starter-tomcat"
     }
-    compile externalDependency.springBootStarterJetty
-    compile externalDependency.springKafka
+    implementation externalDependency.springBootStarterJetty
+    implementation externalDependency.springKafka
+    implementation spec.product.pegasus.restliDocgen
+    implementation spec.product.pegasus.restliSpringBridge
+    implementation externalDependency.slf4jApi
+    implementation externalDependency.log4j2Api
+    compileOnly externalDependency.lombok
+    implementation externalDependency.logbackClassic
+
+    runtime externalDependency.mariadbConnector
+    runtime externalDependency.mysqlConnector
+    runtime externalDependency.postgresql
+
+    annotationProcessor externalDependency.lombok
+
+    testImplementation externalDependency.springBootTest
+    testCompile externalDependency.mockito
+    testCompile externalDependency.testng
 }
 
 bootJar {
     mainClassName = 'com.linkedin.metadata.kafka.MceConsumerApplication'
+    requiresUnpack '**/restli-servlet-impl.jar'
```

Comment:
@RyanHolstien - You'll be interested in this, it is the best solution I've found for working around the restli error when it tries to open a jar within the spring boot fat jar.